packages/esm-reports-app/src/components/next-report-execution.component.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import * as cronjsParser from '@datasert/cronjs-parser';
+import * as cronjsMatcher from '@datasert/cronjs-matcher';
+import moment from 'moment';
+
+interface NextReportExecutionProps {
+  schedule: string;
+  currentDate: Date;
+}
+
+const NextReportExecution: React.FC<NextReportExecutionProps> = ({ schedule, currentDate }) => {
+  const nextReportExecutionDate = (() => {
+    if (!schedule) {
+      return '';
+    }
+
+    const expression = cronjsParser.parse(schedule, { hasSeconds: true });
+    const nextExecutions = cronjsMatcher.getFutureMatches(expression, {
+      startAt: currentDate.toISOString(),
+      matchCount: 1,
+    });
+    return nextExecutions.length == 1 ? moment.utc(nextExecutions[0].toString()).format('YYYY-MM-DD HH:mm') : '';
```

Comment:
We maintain a slew of date handling functions [here](https://github.com/openmrs/openmrs-esm-core/blob/main/packages/framework/esm-utils/src/omrs-dates.ts#L34). They primarily use `dayjs` hence the suggestion above.

Reply:
fixed